common/kafka/config.go

```diff
@@ -28,22 +25,12 @@ type Configuration struct {
 	// Version is the version of Kafka we assume to work
 	Version Version
 	// TLS defines TLS configuration
-	TLS TLSConfiguration
+	TLS TLSAndSASLConfiguration
 }
 
-// TLSConfiguration defines TLS configuration.
-type TLSConfiguration struct {
-	// Enable says if TLS should be used to connect to brokers
-	Enable bool `validate:"required_with=CAFile CertFile KeyFile Username Password SASLAlgorithm"`
-	// Verify says if we need to check remote certificates
-	Verify bool
-	// CAFile tells the location of the CA certificate to check broker
-	// certificate. If empty, the system CA certificates are used instead.
-	CAFile string // no validation as the orchestrator may not have the file
-	// CertFile tells the location of the user certificate if any.
-	CertFile string `validate:"required_with=KeyFile"`
-	// KeyFile tells the location of the user key if any.
-	KeyFile string
+// TLSAndSASLConfiguration defines TLS configuration.
+type TLSAndSASLConfiguration struct {
+	helpers.TLSConfiguration `mapstructure:",squash"`
```

Comment:
You need also `yaml:",inline"` (as the orchestrator may republish it as YAML).

Reply:
oh, missed that :) 

done!